src/mui-file-input.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'mui-file-input';
```

Comment:
Knihovna `mui-file-input` není otypovaná, a proto to řvalo. Toto je workaround pro to, abychom v souboru `src/components/forms/common/FileUploader.tsx` nemuseli mít `// @ts-ignore`.

Reply:
To je zvláštní: https://viclafouch.github.io/mui-file-input/docs/typescript/